src/Symfony/Component/Security/Http/Firewall/SwitchUserListener.php

```diff
@@ -187,6 +214,6 @@ private function getOriginalToken(TokenInterface $token)
             }
         }
 
-        return false;
+        return;
```

Comment:
This is not useless. Token returned by `getOriginalToken()` is compared to `null`. 

However, `@return` doc needs to be updated too.